test-js/unit/pentaho/visual/base/View.Spec.js

```diff
@@ -0,0 +1,57 @@
+define([
+  "pentaho/visual/base/View"
+], function(View) {
+  "use strict";
+
+  describe("pentaho/visual/base/View", function() {
+    var element, dataSpec;
+
+    beforeEach(function() {
+      element = document.createElement("div");
+      var data = {};
+      dataSpec = {v: data};
+    });
+
+    describe("the constructor ", function() {
+
+      it("should throw if not invoked with two arguments", function() {
+        expect(function() {
+          return new View();
+        }).toThrowError(/required/);
+
+        expect(function() {
+          return new View(element);
+        }).toThrowError(/required/);
+
+        expect(function() {
+          return new View(element, dataSpec);
```

Comment:
So I might not have been clear here. What I meant is that you should not pass something other than a model object into the second argument: new Model().